build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1029,6 +1177,44 @@ message ActionResult {
 
   // The details of the execution that originally produced this result.
   ExecutedActionMetadata execution_metadata = 9;
+
+  // The inlined contents of a
+  // [ConsoleOutputs][build.bazel.remote.execution.v2.ConsoleOutputs] message
+  // containing a series of stdout and stderr outputs in order of write from
+  // the action. The server SHOULD NOT inline `console_outputs` unless
+  // requested by the client in the
+  // [GetActionResultRequest][build.bazel.remote.execution.v2.GetActionResultRequest]
+  // message. The server MAY omit inlining, even if requested, and MUST do so if inlining
+  // would cause the response to exceed message size limits.
+  bytes console_outputs = 13;
+
+  // The digest for a blob containing the
+  // [ConsoleOutputs][build.bazel.remote.execution.v2.ConsoleOutputs] for the action,
+  // which can be retrieved from the
+  // [ContentAddressableStorage][build.bazel.remote.execution.v2.ContentAddressableStorage].
+  Digest console_outputs_digest = 14;
+}
+
+// A sequence of console outputs in the order received for a remoted process.
+// This may contain the interleaved outputs from a process in the order
+// received during execution. If the service does not implement
+// interleaving, this may contain a summary of the stdout and stderr output
+// streams as single large writes in a server-defined order.
+message ConsoleOutputs
+{
+  // A series of stdout and stderr outputs in order of write from the remoted process.
+  repeated ConsoleOutput ConsoleOutputs = 2;
```

Comment:
>ConsoleOutputs [](start = 25, length = 14)

Should this be written as `console_outputs` to follow the convention in this `proto` file?

Reply:
Yes! Good catch. Fixed here and several other PascalCase issues.

---
In reply to: [459006683](https://github.com/erikma/remote-apis/pull/1#discussion_r459006683) [](ancestors = 459006683)